node/node.go

```diff
@@ -461,3 +472,13 @@ func (node *Node) AddBeaconChainDatabase(db ethdb.Database) {
 	node.beaconChain = chain
 	node.BeaconWorker = worker.New(params.TestChainConfig, chain, node.Consensus, pki.GetAddressFromPublicKey(node.SelfPeer.ConsensusPubKey), node.Consensus.ShardID)
 }
+
+func (node *Node) InitBlockChainFromDB(db ethdb.Database, consensus *consensus.Consensus) (*core.BlockChain, error) {
+	chainConfig := params.TestChainConfig
+	if consensus != nil {
+		chainConfig.ChainID = big.NewInt(int64(consensus.ShardID)) // Use ChainID as piggybacked ShardID
+	}
+	cacheConfig := core.CacheConfig{Disabled: false, TrieNodeLimit: 256 * 1024 * 1024, TrieTimeLimit: 5 * time.Minute}
```

Comment:
please use some constants here.

Reply:
This is the default value from Ethereum code. They didn't use constant either. This is a placeholder. After we add the archival node role, I will remove it and add config for archival node.